chain/chain/src/chain.rs

```diff
@@ -579,6 +582,7 @@ impl Chain {
             pending_state_patch: Default::default(),
             requested_state_parts: StateRequestTracker::new(),
             state_snapshot_helper: None,
+            state_split_config: StateSplitConfig::default(),
```

Comment:
The view client doesn't need to have resharding configured so it's fine to use default here.